.gitignore

```diff
@@ -5,3 +5,5 @@ build
 .vscode
 __pycache__
 /tests/packages/**/cache
+/tests/packages/**/in/*.in
+/tests/packages/**/out/*.out
```

Comment:
Why do we ignore `in` and `out` files? Aren't there some situations where there might be useful in a test package?

Reply:
I added this because I was hand testing in `abc` package and created the in/out files. And since I was lazy, instead of deleting these files I added them to .gitignore. I'll delete them now.

Reply:
Removed from .gitignore: 2961047328dd96be34def8f089f30e9738dcaa7f